### PR TITLE
Revamp the homepage

### LINKED ISSUE #3: A  proposed template for the homepage
The landing page of the website can be made keeping the following order in mind:
- [x]  Motivation behind the course
- [x]  Overview of the course
- [x]  Lectures in the course
- [ ]  Related Resources 
- [ ]  Assignments

Course Calendar, Staff to remain separate.

### PR DESCRIPTION
**Pull request by srijanpal1405**
"About" section is brought to landing page(i.e. homepage)
cc: @Dsantra92 
issue: #3

### CODE REVIEW

**Comment by 0xf0f00f** [comment]
@forkbabu I will merge after your review

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
 source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
+
+gem "webrick", "~> 1.7"
```
Comment (ucalyptus2) [inline Gemfile:4]: eta add keno kora hyeche?

**Review by ucalyptus2 — COMMENTED** [review]
Can someone show me how this looks like? Ideally the PR can be deployed but I guess a screenshot would do.

File: Gemfile (hunk 1 above)
Comment (0xf0f00f) [reply Gemfile:4]: Dependency break korchilo.

**Comment by ucalyptus2** [comment]
@Fork-chan changes will be reflected when commits come into the main branch.

**Comment by 0xf0f00f** [comment]
Squash merge to main should do the trick.

**Pull request merged**